Doc/using/cmdline.rst

```diff
@@ -994,3 +994,71 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory containing the input
+script (or the the current directory when no file is specified).
+
+On Windows a default Python installation adds the :file:`/Lib` directory and
+the :file:`/Lib/site-packages` directory to the module search path. These
+locations are relative to the :file:`python.exe` location. Standard modules
```

Comment:
I'm not sure if it's helpful or not, but technically the above paths are relative to `prefix` just like on POSIX, where `prefix` is the directory containing `python.exe` (or in a venv, the directory containing the `Scripts` directory which contains `python.exe`).

If we define `prefix` and `exec_prefix` first and describe how each platform finds them (including the `PYTHONHOME` override), maybe it's easier to follow the rest in terms of those values?

Unfortunately, I don't think anyone knows the real difference between `prefix` and `exec_prefix` - maybe this discussion can help flush them out :)

Reply:
I'm not sure either. My aim was to make the new section as brief as possible. A simple overview with pointers to common ways sys.path initialization can be customised. So I don't want the new section to read like a reference document. Introducing `prefix` and `exec_prefix` when describing Windows is confusing, I have a Windows background and had never heard of `prefix `and `exec_prefix `before working on these doc changes. I think the best approach is an overview in the new section with links to detailed notes per platform? Your Windows notes are great, the Unix notes seem less comprehensive and the Mac notes are lacking.

Reply:
I'm working on some improvements. After giving my changes a fresh read today and rereading the comments to date I can see improvements are needed.

Reply:
You've made the biggest start anyone has ever attempted though! Was always going to take a few passes to get it sounding good.

Reply:
> I have a Windows background and had never heard of prefix and exec_prefix before working on these doc changes.

I believe it, but `sys.prefix` and `sys.exec_prefix` are both set and are both useful on Windows (arguably moreso than on other OSs), so introducing them doesn't hurt I don't think.

Combining the stuff that's common into one place and then calling out for platform-specific oddities (which over time should be fixed and then won't have to be documented anymore) is the way I'd like to see this go. But as you say, the challenge is avoiding having it sound like a reference doc!